projects/plugins/jetpack/modules/masterbar/inline-help/inline-help-template.php

```diff
@@ -0,0 +1,12 @@
+<?php
+    // phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+
+	// See https://github.com/Automattic/jetpack/blob/41fe374f32d0fa35bb66b4ca492e0b337ee7c6cd/projects/packages/tracking/src/js/tracks-ajax.js#L29
+	$jp_tracking_classname = 'jptracks';
```

Comment:
Any reason for keeping this on a separate variable instead of directly hard-coding it in the HTLM below?

Reply:
So I could easily comment it to make sure it's obvious that it's required for tracking purposes and not for styling.

So many times in my career have I seen people remove such classes or accidentally break them. Adding this way makes it self documenting and harder to break.

Personal preference though...

Reply:
Incidentally, this is why I prefer `data-` attributes for such things as opposed to classes.

Reply:
Why do we even need the classname if we have a `data-jptracks-` which we could use to select things in the DOM with a minimal perf penalty?